components/contribution-flow/index.js

```diff
@@ -483,28 +491,36 @@ class CreateOrderPage extends React.Component {
   }
 
   getLoggedInUserDefaultContibuteProfile() {
+    const { LoggedInUser } = this.props;
+
     if (get(this.state, 'stepProfile')) {
       return this.state.stepProfile;
+    } else if (this.props.contributeAs) {
```

Comment:
@kewitz in the previous version `this.props.contributeAs` had priority over `this.state.stepProfile`. Can you double check this is what we want?

Reply:
It doesn't make a difference in this context since this runs before `this.state.stepProfile` is being set.
But I agree it is better to keep the priority, it is more explicit about the intent of having contributeAs overriding any other value if possible.